tests/test_auth.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from requests_futures.sessions import FuturesSession
+
+from harmony_py.auth import _authenticate, _is_edl_hostname, authenticate, BadAuthentication, \
+                            MissingCredentials, SessionWithHeaderRedirection
+
+#
```

Comment:
Remove commented out code in this file.

Reply:
Those comments were there to convey the test organization to the reader but I can remove them.